plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/TestHudiSmokeTest.java

```diff
@@ -659,21 +657,13 @@ public void testDynamicFilterDisabledPredicatePushdown(ResourceHudiTablesInitial
         MaterializedResult explainRes = getQueryRunner().execute(session, "EXPLAIN ANALYZE " + query);
         Pattern scanFilterInputRowsPattern = getScanFilterInputRowsPattern(tableIdentifier);
         Matcher matcher = scanFilterInputRowsPattern.matcher(explainRes.toString());
+        // Should not find any dynamic filtering keywords
         assertThat(matcher.find())
                 .withFailMessage("Could not find 'ScanFilter' for table '%s' with 'dynamicFilters' and 'Input: X rows' stats in EXPLAIN output.\nOutput was:\n%s",
                         tableIdentifier, explainRes.toString())
-                .isTrue();
-
-        // matcher#group() must be invoked after matcher#find()
-        String rowsInputString = matcher.group(1);
-        long actualInputRows = Long.parseLong(rowsInputString);
-        // No dynamic filtering performed, all rows read
-        long expectedInputRowsAfterFiltering = 4;
-        assertThat(actualInputRows)
-                .describedAs("Number of rows input to the ScanFilter for the probe side table (%s) should NOT reflect effective dynamic filtering", tableIdentifier)
-                .isEqualTo(expectedInputRowsAfterFiltering);
+                .isFalse();
```

Comment:
Is there a bug invalidation before?

Reply:
Sorry, modified this by mistake. let me fix it.